pyo3-build-config/src/impl_.rs

```diff
@@ -825,7 +825,7 @@ pub(crate) fn cross_compile_env_vars() -> CrossCompileEnvVars {
 ///
 /// See the [PyO3 User Guide](https://pyo3.rs/) for more info on cross-compiling.
 #[deprecated(
-    since = "0.17.0",
```

Comment:
I usually am cautious about deprecating in patch releases, however I think it's so unlikely that this is used widely that it's fine.